packages/react/src/components/login/Login.stories.tsx

```diff
@@ -442,6 +465,59 @@ const NotifyIfErrorOccurs = () => {
   );
 };
 
+const ProfileData = () => {
```

Comment:
Above ProfileData is replaced with UserData, should it be here as well or are these different?